app/views/search/_result_empty.html.erb

```diff
@@ -1 +1 @@
-<li>There are no results.</li>
+<h2>No results found for your search</h2>
```

Comment:
I had a weird a11y moment yesterday when I realized that 'no results' isn't focusable at all, either here or in prod. Presumably this is because it's not interactive, but it probably should be in some way? I didn't touch it here since it felt like another ticket, and I figured I was missing something due to my lack of screen reader knowledge.

Reply:
That may partially be your browser settings to only tab to actionable items. Hopefully.

Reply:
Hopefully. 🤷 In the meantime, I opened this ticket to address the tab order: https://mitlibraries.atlassian.net/browse/GDT-161